flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolTest.java

```diff
@@ -599,18 +603,17 @@ public void testCheckIdleSlot() throws Exception {
 	}
 
 	/**
-	 * Tests that idle slots which cannot be released are only recycled if the owning {@link TaskExecutor}
-	 * is still registered at the {@link SlotPoolImpl}. See FLINK-9047.
+	 * Tests that idle slots which cannot be released will retry until the exception should be ignored. See FLINK-11059.
 	 */
 	@Test
 	public void testReleasingIdleSlotFailed() throws Exception {
```

Comment:
The test is no longer correct. It assumes that `TaskExecutorGateway#freeSlot` is called twice whereas it should only be called once.

Reply:
Yes, I refine the tests